src/js/remoteServicesCommunity.js

```diff
@@ -213,4 +213,37 @@ export default {
     projectId: (document) => projectIdentifierBySelector(".current-project")(document),
     allowHostOverride: true,
   },
+
+  // visualstudio.com/azure devops service configs provided by lx-media (@barokai)
+  visualstudiocom: {
+    name: "visualstudiocom",
+    host: "https://:org.visualstudio.com",
+    // https://org.visualstudio.com/project/_workitems/edit/id/
+    urlPatterns: [":host:/:project/_workitems/edit/:id(/*)"],
+    description: (document, _service, { title }) => {
+      console.log("trying to load description information for azuredevopsvs");
+      const issueIdMatch = window.location.href.match(/_workitems\/edit\/(\d+)(?:\/.*)?(?:\?.*)?/)
+      const issueId = issueIdMatch ? issueIdMatch[1] : null
+      const issueTitle = document.querySelector(".work-item-title-textfield>input")?.value.trim() || title
+      return issueId ? `${issueId}: ${issueTitle}` : issueTitle
+    },
```

Comment:
The issue ID should already be passed as parameter to the description callback, `(document, _service, { id, title }) => {

Reply:
Thank you for the comments. I've updated the code accordingly.

The matcher was used to cover edge cases like a query string directly after the ID, but pretty sure not needed at all; therefore changed to the recommended handling.
Removed title parameter as but using project name now as fallback.


Sadly, I encountered a problem during testing:
When trying to submit the activity via createActivity, an error with status code 422 is caught: 
`error.response.data` only contains `remote_service: ["ist kein gültiger Wert"]`.
`remote_service` in the activity object is correctly set as `"visualstudiocom"` when this happens.
Do you maybe have any idea about that?